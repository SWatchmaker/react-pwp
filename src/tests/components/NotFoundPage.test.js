import React from "react";
import NotFoundPage from "../../components/NotFoundPage";
import { shallow } from "enzyme";

test("Renderiza pÃ¡gina no encontrada", () => {
  const wrapper = shallow(<NotFoundPage />);
  expect(wrapper).toMatchSnapshot();
});
