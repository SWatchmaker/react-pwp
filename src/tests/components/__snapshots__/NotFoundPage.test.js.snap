// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renderiza página no encontrada 1`] = `
<div>
  404! - 
  <Link
    replace={false}
    to="/"
  >
    Ir a página principal
  </Link>
</div>
`;
